deploy/signup-with-plugins/signup-with-mock-plugin/values.yaml

```
@@ -12,6 +12,8 @@ extraEnvVars:
         key: mosip-esignet-host
       configMapKeyRef:
```

YAMLlint (1.37.1)

[error] duplication of key "configMapKeyRef" in mapping

(key-duplicates)